schemas/context/account-crm.example.1.json

```diff
@@ -0,0 +1,95 @@
+{
+  "xdm:ownerID": "OWNER111",
+  "xdm:parentID": "ACC000",
+  "xdm:partnerAccountID": "PRTNR1111",
+  "xdm:masterRecordID": "MSTR111",
+  "xdm:territoryID": "Terr1",
+  "xdm:accountSite": "Additional account site details",
+  "xdm:currencyCode": "USD",
+  "xdm:annualRevenue": 10000000,
+  "xdm:fax": [
+    {
+      "xdm:primary": true,
+      "xdm:number": "1-408-888-8888",
+      "xdm:status": "active"
+    }
+  ],
+  "xdm:accountOrganization": {
+    "@id": "https://data.adobe.io/entities/org/orgid111",
+    "xdm:legalName": "ABC Stores",
+    "xdm:numberOfEmployees": 200,
+    "xdm:industry": "Retail",
+    "xdm:website": "http://www.abcstores.com",
+    "rating": 1,
+    "xdm:marketSegment": "apparel for kids",
+    "xdm:location": {
+      "@id": "https://data.adobe.io/entities/places/locid123",
+      "schema:description": "ABC Store location",
+      "xdm:address": {
+        "@id": "https://data.adobe.io/entities/address/123",
+        "xdm:primary": true,
+        "xdm:type": "work",
+        "xdm:street1": "100 Lincoln Ave",
+        "xdm:city": "San Jose",
+        "xdm:stateProvince": "US-CA",
+        "xdm:postalCode": "95125",
+        "xdm:country": "United States",
+        "xdm:countryCode": "US",
+        "schema:latitude": 37.3382,
+        "schema:longitude": 121.8863,
+        "xdm:status": "active",
+        "xdm:lastVerifiedDate": "2018-01-03"
```

Comment:
modify the "dd" (> 12) to differentiate mm

Reply:
These examples were taken from existing Schema examples. Those need to be fixed as well. I will fix this one.